pom.xml

```diff
@@ -215,7 +215,6 @@
                         <artifactId>junit-jupiter-engine</artifactId>
                         <version>5.2.0</version>
                     </dependency>
-
```

Comment:
It looks like there's just an accidental whitespace change here.  If pom.xml isn't really modified, it shouldn't be part of the PR.